src/docs/development/add-to-app/ios/add-flutter-screen.md

```diff
@@ -0,0 +1,487 @@
+---
+title: Adding a Flutter screen to an iOS app
+short-title: Add a Flutter screen
+description: Learn how to add a single Flutter screen to your existing iOS app.
+---
+
+This guide describes how to add a single Flutter screen to an existing iOS app.
+
+## Integration Steps
+
+ 1. [Create Flutter Module](#create-a-flutter-module) - This creates the Flutter project which will be hosted in the existing app.
+ 1. [Add Module Dependency in Existing App](#make-the-host-app-depend-on-the-flutter-module) - This sets up XCode to include your Flutter project into the build of your existing app.
+ 1. [Add FlutterViewController to Existing App](#write-code-to-use-flutterviewcontroller-from-your-host-app) - This steps introduces the code that will conjure the instance of Flutter into your existing app.
+ 1. (optional) [Attach Flutter Tool](#building-and-running-your-app) - This step allows you to attach to Flutter's debugger and enables you do use hot reload / restart.
+
+## Create a Flutter module
+
+Flutter projects created using `flutter create xxx` include very simple host
+apps for your Flutter/Dart code (a single-ViewController iOS host). You can
+modify these host apps to suit your needs and build from there.
+
+But if you're starting off with an *existing* host app for either platform,
+you'll likely want to include your Flutter project in that app as some form of
+library instead.
+
+This is what the Flutter module template provides. Executing
+`flutter create -t module xxx` produces a Flutter project containing a CocoaPods
```

Comment:
Do we need both sentences? Maybe the previous sentence can be deleted?

Reply:
My concern with deleting it is that it is introducing the terminology "flutter module template" . I'm not sure if that is used in other documents.